server/proxy/pf_client.c

```diff
@@ -90,6 +91,15 @@ static BOOL pf_client_end_paint(rdpContext* context)
 	return ps->update->EndPaint(ps);
 }
 
+static BOOL pf_client_remote_monitors(rdpContext* context, UINT32 count, const MONITOR_DEF* monitors)
+{
+	pClientContext* pc = (pClientContext*) context;
+	proxyData* pdata = pc->pdata;
+	rdpContext* ps = (rdpContext*)pdata->ps;
+
+	return freerdp_display_send_monitor_layout(ps);
```

Comment:
We don’t pass the monitors we got at all :(((

Reply:
We might want to update the settings as well

Reply:
lol i'll fix it sorry